doc/api/topics/index_-_ok_-_without_params.json

```diff
@@ -0,0 +1,46 @@
+{
+  "resource": "Topics",
+  "resource_explanation": null,
+  "http_method": "GET",
+  "route": "/api/v1/topics",
+  "description": "Index - OK - Without params",
+  "explanation": null,
+  "parameters": [
+
+  ],
+  "response_fields": [
+
+  ],
+  "requests": [
+    {
+      "request_method": "GET",
+      "request_path": "/api/v1/topics",
+      "request_body": null,
+      "request_headers": {
+        "Access-Token": "H5DVUqDOCXK8eRD652Jgpw",
```

Comment:
Is this a real token? you should generate the documentation again so it's replaced with the you hardcoded in [spec/support/devise_token_auth_overrides.rb](https://github.com/LeandroSoria2/TargetApi/pull/21/files#diff-e55b07c29170e3b2b4d831524ea60ebaab4acf0772d361e8909a37195bed12f9)

Reply:
It should look something like this:

<img width="336" alt="imagen" src="https://user-images.githubusercontent.com/80283482/180240870-7dd5b90c-cf01-4ae6-b128-67bfa7407582.png">

Reply:
Thanks, I will change it on the next pr